app/.env.docker-example

```diff
@@ -7,12 +7,12 @@ DJANGO_SUPERUSER_EMAIL=admin@admin.com
 DJANGO_SUPERUSER_PASSWORD=admin
 
 # postgres settings for docker
-SQL_ENGINE=django.db.backends.postgresql
-SQL_DATABASE=people_depot_dev
-SQL_USER=people_depot
-SQL_PASSWORD=people_depot
-SQL_HOST=db
-SQL_PORT=5432
+POSTGRES_ENGINE=django.db.backends.postgresql
+POSTGRES_DATABASE=people_depot_dev
+POSTGRES_USER=people_depot
+POSTGRES_PASSWORD=people_depot
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
```

Comment:
I see there are other places with the "SQL_ENGINE" type of names. You should change them as well. They're commented out, but they're also examples for non-default cases.

This relates to what I said about whether or not changing the variable names make sense. I think if we're not going to run the backend with sqlite anymore, changing the name makes sense. If we want to continue making the backend work with sqlite, then maybe it doesn't make sense to change the variable names to have the "POSTGRES_" prefix. In that case, you can add the "POSTGRES-*" variables and do something like "SQL_USER=$POSTGRES_USER".